### PR TITLE
update shippable config

### LINKED ISSUE #1793: Shippable failed when building to release branch
Today when making a pr to `b0.10.0` i noticed Shippable failed to merge develop into my branch:
https://app.shippable.com/github/golemfactory/golem/runs/2147/summary/console

Maybe we can configure the "base-branch" or take it from the PR to get the right target branch to compare against

### PR DESCRIPTION
**Pull request by etam**
Fixes #1793

### CODE REVIEW

File: shippable.yml [hunk 1]
```diff
@@ -3,12 +3,14 @@ language: python
 python:
   - 3.5
 
-install:
-  - pip3 install -r requirements.txt
-  - pip3 install -r requirements-lint.txt
-
-script:
-  - git checkout -b shippable-lint  # Shippable doesn't make a branch for us, unlike CircleCI
-  - git merge origin/develop        # Update the changes from develop to avoid false positives
```
Comment (maaktweluit) [inline shippable.yml:12]: Removing the merge of the base_branch will give false positives some times when the PR branch is not updated to develop properly.

**Review by maaktweluit — CHANGES_REQUESTED** [review]
LGTM!

Nice improvement

One small q, can approve on reply.

File: shippable.yml (hunk 1 above)
Comment (etam) [reply shippable.yml:12]: http://docs.shippable.com/ci/pull-request-builds/

>  This is done by pulling the latest code from your your branch, i.e. the branch where the pull request will be merged, merging with the changes in the pull request, and running your build against the merged code.

Comment (maaktweluit) [reply shippable.yml:12]: TIL... :D

In this case we only need it on buildbot since its not as cool as shippable :)

**Review by maaktweluit — APPROVED** [review]

**Pull request merged**